ttnn/cpp/ttnn/operations/conv2d/conv2d_pybind.hpp

```diff
@@ -55,6 +55,65 @@ void py_module(py::module& module) {
         py::arg("bias_tensor") = std::nullopt,
         py::arg("conv_config") = std::nullopt);
 
+    module.def(
+        "optimized_conv",
+        &optimized_conv,
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg("bias").noconvert() = std::nullopt,
+        py::arg("conv_reader_indices").noconvert() = std::nullopt,
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert() = 0,
+        py::arg("output_mem_config").noconvert() = std::nullopt,
```

Comment:
`memory_config` for consistency with other ttnn operations

Reply:
Done